UpdateSubscriptionsFeedActivity/index.ts

```diff
@@ -0,0 +1,176 @@
+﻿import * as crypto from "crypto";
```

Comment:
Do we really want to maintain all this logic when we just want to add an UNSUBSCRIBE event in the table?

Reply:
May you mark the parts that you think are redundant? I've asked the same question myself, then I noticed that the intent here was to have a general purpose activity (as the name is indeed 'UpdateSubscriptionsFeedActivity' and not 'UnsubscribeUser'). If we switch this to `UnsubscribeUser` we still need to keep 80% of the logic and you only gain about 10 lines less getting rid of the 'subscribe' branch.

Reply:
The risk of duplicating the "subscribe" code here is that you must maintain it in sync if changes happens.
About sharing the code inside io-functions-commons i am more convinced that subscriptions must be managed by a specific/separate service.

Reply:
what do you suggest in the meantime?

Reply:
It is ok for me to merge as is.